TaskVlopper/Controllers/MeetingController.cs

```diff
@@ -184,9 +185,11 @@ public ActionResult Users(int id)
                 if (User.Identity.IsAuthenticated)
                 {
                     IMeetingLogic logic = container.Resolve<IMeetingLogic>();
-                    var viewmodel = logic.GetMeetingUsers(id).ToList();
+                    var queryMeetingUsers = logic.GetMeetingUsers(id);
 
-                    return Json(viewmodel, JsonRequestBehavior.AllowGet);
+                    var viewModel = new UsersViewModel();
+                    viewModel.Users.AddRange(queryMeetingUsers.Select(x => new UserViewModel(x)));
```

Comment:
I've added overloaded constructor that can do AddRange. In ProjectViewModel but that's a baseline mostly